README.md

```diff
@@ -6,4 +6,4 @@
 | 2017 | Andross (recommended)  | React Navigation, Redux, Redux Sagas          | https://github.com/infinitered/ignite-ir-boilerplate-andross  |
 | 2016 | Ignite 1.0 | React Native Router Flux, Redux, Redux Sagas | https://github.com/infinitered/ignite-ir-boilerplate-2016 |
 
-To support legacy versions of Ignite, https://www.npmjs.com/package/ignite-ir-boilerplate will continue to deliver our currently reccomended boilerplate, but we recommend updating Ignite CLI so you can manually select which boilerplate you would like to use.
+To support legacy versions of Ignite, https://www.npmjs.com/package/ignite-ir-boilerplate will continue to deliver our currently recomended boilerplate, but we recommend updating Ignite CLI so you can manually select which boilerplate you would like to use.
```

Comment:
```suggestion
To support legacy versions of Ignite, https://www.npmjs.com/package/ignite-ir-boilerplate will continue to deliver our currently recommended boilerplate, but we recommend updating Ignite CLI so you can manually select which boilerplate you would like to use.
```

Reply:
Be sure to spell "recomended" as "recommended" in the above change request.